### DISCUSSION TITLE
Web based Jabref project ?

### DISCUSSION THREAD

**Discussion by Pl1997** [JabRef/JabRefOnline#1474] [General]
Hello !

I had understood that some development was ongoing regarding a web based version of JabRef, which I was highly anticipating despite not being skilled enough to contribute to such a project…

I’m under the impression that it now has been abandoned ? (Since the « project » page was closed)

Would you be so nice to share some info about this ?

Thank you very much

**Comment by tobiasdiez** [comment]
Thanks for your interest. Yes, the aim of this project is to provide a web platform to interact with your bibliographic data (and of course sync it to your desktop JabRef). It's still at an early stage but you can login in the test account using alice@jabref.org / EBNPXY35TYkYXHs at https://mango-pebble-0224c3803-dev.westeurope.1.azurestaticapps.net/user/login to see the current state.

We don't have the resources for a proper user study, but its of course super helpful to know what our users expect from a "JabRef in the Web" platform. So if you have a few spare minutes and would like to support these efforts, could you shortly describe what features you would like to see, how you would interact with the web version (why is the desktop not enough for your workflow?) and in which direction you would take the project if you could dream and everything would be possible. Thanks!

**Comment by Pl1997** [comment]
Thanks for your answer ! I'm so happy that this project is still ongoing. Here are the main things that make me interested : 
- I use different devices on a daily basis, including an iPad. Since Jabref doesn't have a mobile app yet, it would be very useful to get a cross-platform access to my library (including some online access to the full documents)
- I hope that this web-based Jabref can work with groups : it would allow me to share documents or specific folders in my library with collaborators.

Currently, Zotero could answer my needs, but you have to pay to get some proprietary online storage (mandatory for group libraries, and not compatible with any cross platform protocol such as webdav). I would much prefer a self hosted, fully open source solution !

**Comment by tobiasdiez** [comment]
Thanks for your feedback! Very reassuring, as indeed both of these features are planned.

**Comment by wangyenshu** [comment]
Hello. Is the web based jabref project still in progress? The url https://mango-pebble-0224c3803-dev.westeurope.1.azurestaticapps.net/user/login returns 404.

The old version of jabref can run in browser via cheerpj: https://wangyenshu.github.io/jabref-in-browser/.
It would be so nice if the latest version of jabref can be used in browser. 
<img width="1484" height="900" alt="Screenshot_29" src="https://github.com/user-attachments/assets/eb7d39a3-0df9-4986-831a-6df5a77960d1" />
